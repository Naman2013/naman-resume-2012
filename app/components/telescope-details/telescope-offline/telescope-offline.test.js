import React from 'react';
import renderer from 'react-test-renderer';
import TelescopeOffline from './telescope-offline';

it('should render with an image and message', () => {
  const tree = renderer.create(
    <TelescopeOffline
      imageSource="https://vega.slooh.com/images/placeholders/telescope-offline.png"
      offlineStatusMessage="Missions are offline due to weather conditions."
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
