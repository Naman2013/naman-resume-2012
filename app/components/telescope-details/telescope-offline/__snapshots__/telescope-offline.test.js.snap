// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with an image and message 1`] = `
<div
  className="jsx-2181730630 telescope-details-offline"
>
  <h2
    className="jsx-2181730630"
  >
    OFFLINE
  </h2>
  <div
    className="jsx-2181730630"
  >
    <img
      alt="Telescope is currently offline"
      className="jsx-2181730630"
      height="350"
      src="https://vega.slooh.com/images/placeholders/telescope-offline.png"
    />
  </div>
  <div
    className="jsx-2181730630 offlineMessage"
  >
    <h5
      className="jsx-2181730630"
    >
      Missions are offline due to weather conditions.
    </h5>
  </div>
</div>
`;
